mathphys/functions.py

```diff
@@ -10,7 +10,9 @@
 from types import ModuleType as _ModuleType
 import gzip as _gzip
 
-import h5py as _h5py
+if _importlib.util.find_spec('h5py') is not None:
+    import h5py as _h5py
```

Comment:
Isn't it more idiomatic to do `try: import h5py ...`?

The result is the same, we successfully import `h5py` and create the `_h5py` symbol if the dependency is available, but it uses builtin try-except instead.